src/web/lib/components/Banner/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import './index.scss';
+
+export const Banner = ({ addonUrl, bottom = false }) =>
```

Comment:
I thought we might want to play around with the placement of the banner so i made optionally fixed to the bottom of the page